test/generators/serializer_generator_test.rb

```diff
@@ -20,11 +20,10 @@ def test_generates_a_namespaced_serializer
   end
 
   def test_uses_application_serializer_if_one_exists
-    Object.const_set(:ApplicationSerializer, Class.new)
-    run_generator
-    assert_file 'app/serializers/account_serializer.rb', /class AccountSerializer < ApplicationSerializer/
-  ensure
-    Object.send :remove_const, :ApplicationSerializer
+    stub_safe_constantize(expected: 'ApplicationSerializer') do
```

Comment:
`safe_constantize` would return `nil`, meaning the generator would (incorrectly) inherit from `ActiveModel::Serializer` instead of `ApplicationSerializer`:

```
  1) Failure:
SerializerGeneratorTest#test_uses_application_serializer_if_one_exists [/Users/lrichmond1/code/active_model_serializers/test/generators/serializer_generator_test.rb:25]:
Expected /class AccountSerializer < ApplicationSerializer/ to match "class AccountSerializer < ActiveModel::Serializer\n  attributes :id, :name, :description\n  has_one :business\nend\n".
```